tests/resources/mlflow-test-plugin/mlflow_test_plugin/app.py

```diff
@@ -23,6 +23,12 @@ def before_req_hook():
 
     Can implement things such as authentication, special handling, etc.
     """
-    app_logger.warning("Hello from before request!")
     if not is_logged_in():
+        app_logger.warning("Hello from before request!")
         return "Unauthorized", 403
```

Comment:
To avoid showing this message when we run `mlflow server` in the root, moved `app_logger.warning("Hello from before request!")`.